lib/awful/placement.lua

```diff
@@ -603,6 +617,150 @@ local function rect_to_point(rect, corner_i, corner_j)
     }
 end
 
+-- Create a pair of rectangles used to set the relative areas.
+-- v=vertical, h=horizontal
+local function get_cross_sections(abs_geo, mode)
+print("\n\nLA!",mode)
```

Comment:
Still a leftover print a week later and when "WIP" is removed. I'll stop reading commits here and wait for you to specifically say "I handled all existing comments, please review again".